spec/policy/upstream_mtls/upstream_mtls_spec.lua

```diff
@@ -119,5 +129,82 @@ describe('Upstream MTLS policy', function()
 
   end)
 
+  describe("CA_Certificates", function()
+
+
+    before_each(function()
+      stub.new(upstream_mtls, 'set_ca_cert', function() return true end)
+    end)
+
+    it("ca_store is nil if no certificates", function()
+      local config = {
+        certificate = "XXXX",
```

Comment:
You could extract this string to a variable accessible from all the tests and reuse it everywhere.